flowmachine/flowmachine/features/location/redacted_labelled_query.py

```diff
@@ -11,45 +11,45 @@
 from flowmachine.core.mixins import GeoDataMixin
 
 
-class RedactedLabelledSpatialAggregate(GeoDataMixin, Query):
+class RedactedLabelledSpatialQuery(GeoDataMixin, Query):
```

Comment:
I think we'll still need a separate class for redacted labelled flows, because that will need the `FlowLike` mixin instead of `GeoDataMixin`. Might be better to use `RedactedLabelledSpatialQuery` as a mixin for `RedactedLabelledSpatialAggregate` and `RedactedLabelledFlows` (similar to `RedactedLocationMetric`, which is a mixin used by several other redacted queries).